src/ray/rpc/client_call.h

```diff
@@ -314,7 +315,11 @@ class ClientCallManager {
                 // The call is finished, and we can delete this tag now.
                 delete tag;
               },
-              std::move(stats_handle));
+              stats_handle->event_name + ".OnReplyReceived",
```

Comment:
Is it possible to test this using e2e test + manually calling grpc stub?

Reply:
Added.